.env.development

```diff
@@ -1,4 +1,4 @@
 VUE_APP_API_BASE_URL=https://mf-chsdi3.int.bgdi.ch/
 VUE_APP_DATA_BASE_URL=https://data.geo.admin.ch/
 VUE_APP_WMTS_BASE_URL=https://wmts{5-9}.int.bgdi.ch/
-VUE_APP_WMS_BASE_URL=https://wms-bgdi-cdn-{0-4}.int.bgdi.ch
+VUE_APP_WMS_BASE_URL=https://wms-bgdi-cdn-0.int.bgdi.ch
```

Comment:
new FQDN for me, how is `wms-bgdi-cdn-0` different from `wms-bgdi` ?

Reply:
That's values I've copy pasted from `rc_int` and `rc_dev` on geoadmin's code

Reply:
ok